SpanishFilter/sections/css_extended.txt

```diff
@@ -11,6 +11,7 @@ forum.lfspro.net#?##ipsLayout_mainArea > section:has(> ol > li > h2 > a:contains
 forum.lfspro.net#?#.ipsList_reset > li.ipsWidget:has(> h3:contains(LFSPro Ads))
 forum.lfspro.net#?##ipsLayout_mainArea > .ipsBox:has(> h2 > a:contains(LFSPro Ads))
 forum.lfspro.net#?##elPostFeed > form > center:has(> ins.adsbygoogle)
+perfil.com#?#div[style]:has(> div.ads-space)
```

Comment:
better `div[style]` to `div[style^="width: 100%;"][style*="align-items: center;"]`